src/Simulation/CsharpGeneration.Tests/SimulationCodeTests.fs

```diff
@@ -1093,7 +1093,7 @@ namespace N1
             {
                 var qubits = Allocate.Apply(3L);
 #line hidden
-                Exception __arg1__ = null; 
```

Comment:
All the tests that I updated check that generated code is equal to the string provided in the text.
I had to update all the tests like this, because new exception handling generates new code.